doc/7/controllers/document/m-upsert/snippets/m-upsert.js

```diff
@@ -0,0 +1,44 @@
+const doc1 = { capacity: 4 };
+const doc2 = { capacity: 7 };
+
+try {
+  await kuzzle.document.create('nyc-open-data', 'yellow-taxi', doc1, 'some-id');
+  await kuzzle.document.create('nyc-open-data', 'yellow-taxi', doc2, 'some-other-id');
```

Comment:
(nitpicking) I would remove that line to make your code update a document, and create another one, all in one go.

```suggestion
```